src/megaapi_impl.cpp

```diff
@@ -19112,6 +19112,7 @@ void MegaApiImpl::sendPendingRequests()
             }
             else
             {
+                committer.commitNow(true);
```

Comment:
We may not want to commit the open transaction in all cases. Remember the `sctable` should only commit upon reception of a new `scsn` via actionpackets (or a bit later when the ongoing `cs` requests complete).

In this case, I think to commit the ongoing transaction is incorrect and the account's state may get corrupted. Thanks

Reply:
Yes you're right that we must be very careful about committing sctable.   This one would only be used for tctable though.